src/backup.ts

```diff
@@ -97,26 +111,36 @@ export type ExistsCallback = NormalCallback<boolean>;
  * const spanner = new Spanner();
  * const instance = spanner.instance('my-instance');
  * const backup = instance.backup('my-backup');
+ * const sourceName = instance.backup('my-source-backup');
  */
 class Backup {
   id: string;
   formattedName_: string;
   instanceFormattedName_: string;
   resourceHeader_: {[k: string]: string};
   request: BackupRequest;
+  sourceName: string | undefined;
   metadata?: databaseAdmin.spanner.admin.database.v1.IBackup;
-  constructor(instance: Instance, name: string) {
+  constructor(instance: Instance, name: string, sourceName?: string) {
     this.request = instance.request;
     this.instanceFormattedName_ = instance.formattedName_;
     this.formattedName_ = Backup.formatName_(instance.formattedName_, name);
     this.id = this.formattedName_.split('/').pop() || '';
+    if (sourceName) {
```

Comment:
nit: I think this `if` statement is redundant. You can just unconditionally assign `sourceName` to `this.sourceName`.